lib/classes/CLI.js

```diff
@@ -67,7 +67,7 @@ class CLI {
     this.consoleLog('');
 
     this.consoleLog(chalk.yellow.underline('Commands'));
-    this.consoleLog(chalk.dim('* Serverless documentation: http://docs.serverless.com'));
```

Comment:
Please remove changing of the docs from here for now. Opened a new issue in #2167 to be able to collect some feedback from the community before we release in the next few days.